### ISSUE TITLE
Taking container to production

### ISSUE THREAD

**Issue by napulen** [DDMAL/cantus#332]
Currently, the cantus production website is a version built "in-place" in ComputeCanada. It didn't make use of the provisioning scripts and vagrant box that we currently have. 

This wasn't a problem for the most part because the vagrant box was trying to replicate that production environment. However, now the vagrant box has moved forward in development and the production version is outdated. 

We need to host the new box in production. This may require looking into deploying vagrant boxes using the hypervisor of our cloud service as the provider of the vagrant box.

**Comment by napulen** [comment]
- [ ] Can vagrant containers be launched in ComputeCanada?

**Comment by napulen** [comment]
They can. Using the [vagrant-openstack-provider](https://github.com/ggiamarchi/vagrant-openstack-provider) I was able to launch the container in Compute Canada. The container works as expected once the instance in Compute Canada has enough disk.

That means there is more work to do to get this container toward a minimal production version that can be deployed in CC. In principle, I have demonstrated that the workflow of using OpenStack as a remote provider works well.
